### DISCUSSION TITLE
Evolution sync fail

### DISCUSSION THREAD

**Discussion by AnkitSavani111** [ROM-builders/temporary#17216] [General]
Facing same error as others
https://cirrus-ci.com/task/6515931023671296?logs=sync#L33
https://cirrus-ci.com/task/4519517783588864?logs=sync#L35

**Comment by anandhan07** [comment]
Not only Evolution, now it's failing for all. It's been 5 days now 🥲. So far no successful build and couldn't release this month's update.
https://cirrus-ci.com/github/ROM-builders/temporary

After yesterday's clean sync: https://cirrus-ci.com/task/4990213181669376

Today's build: https://cirrus-ci.com/task/5875573679980544

![Screenshot_20230123-154812_Brave](https://user-images.githubusercontent.com/79000628/214017566-16e30832-6549-4b7a-a2b7-03fdea1fe9af.png)

**Reply by NRanjan-17** [reply]
here too evo sync issue

**Comment by NRanjan-17** [comment]
https://cirrus-ci.com/task/6515931023671296

Again same @Apon77

**Comment by hipexscape** [comment]
Mine also not working pe and pixel os
